content/file/utils.go

```diff
@@ -61,6 +61,7 @@ func tarDirectory(ctx context.Context, root, prefix string, w io.Writer, removeT
 		name = filepath.ToSlash(name)
 
 		// Generate header
+		// NOTE: We don't support hard links and treat it as regular files
```

Comment:
If there wqs a duplicate blob in this with the same shasum, a hard link would be ideal. Is that a possibility with this method?

Reply:
Do you mean users may push a directory containing two files, one of them is a hark link to another?